src/components/actions/ActionListFilters.tsx

```diff
@@ -1121,30 +1131,48 @@ function ActionListFilters(props: ActionListFiltersProps) {
 }
 
 type ConstructFiltersOpts = {
-  mainConfig: ActionListPageFiltersFragment;
+  mainConfig: ActionListPageFiltersFragment | IndicatorListPageFiltersFragment;
   plan: PlanContextType;
   t: TFunction;
   orgs: ActionListOrganization[];
   primaryOrgs: ActionListPrimaryOrg[];
   filterByCommonCategory: boolean;
   actionTerm?: string;
+  isIndicatorList?: boolean;
 };
-
 ActionListFilters.constructFilters = (opts: ConstructFiltersOpts) => {
```

Comment:
Also, I think you've mentioned it before, but this all ain't very React-y or reactive. Could be part of a wider refactor.